docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -74,7 +74,25 @@ This section MUST be read together with the ABNF section. In case of disagreemen
 
 #### Values
 
-Metric values in OpenMetrics MUST be either floating points or integers. Note that ingestors of the format MAY only support float64. The non-real values NaN, +Inf and -Inf MUST be supported. NaN MUST NOT be considered a missing value, but it MAY be used to signal a division by zero.
+Metric values in OpenMetrics MUST be either Number or ComplexValue.
+
+##### Number
+
+Number value MUST be either floating point or integer. Note that ingestors of the format MAY only support float64. The non-real values NaN, +Inf and -Inf MUST be supported. NaN value MUST NOT be considered a missing value, but it MAY be used to signal a division by zero.
```

Comment:
@bwplotka I've removed the forward reference to the format and made "Number" and "ComplexValue" it's own section to be able to refer to it if we want and follow the convention.